src/storage/redis_db.cc

```diff
@@ -99,10 +99,18 @@ rocksdb::Status Database::GetMetadata(RedisTypes types, const Slice &ns_key, std
 }
 
 rocksdb::Status Database::GetRawMetadata(const Slice &ns_key, std::string *bytes) {
+  return getRaw(ns_key, bytes);
+}
+
+rocksdb::Status Database::GetRawData(const Slice &key, std::string *bytes) {
+  return getRaw(key, bytes);
+}
+
+rocksdb::Status Database::getRaw(const Slice &key, std::string *bytes) {
```

Comment:
Could you explain a little about these newly-added methods?

Reply:
'GetRawData' is added just because need to read 'sub_key' value as the method literal meaning, corresponds to 'GetRawMetadata'.

Reply:
oh, but the column family should be specified, I'll fix it.

Reply:
The code changed here should be reset, it's not in use now.